scripts/model_figs/analyze_model_disagreement.py

```diff
@@ -30,23 +53,29 @@
 # also possible models failed to learn whatever physics makes these materials highly
 # unstable
 
-material_classes = {
-    "all": r".*",
-    "lanthanides": r".*(La|Ce|Pr|Nd|Pm|Sm|Eu|Gd|Tb|Dy|Ho|Er|Tm|Yb|Lu)\d.*",
-    "actinides": r".*(Ac|Th|Pa|U|Np|Pu|Am|Cm|Bk|Cf|Es|Fm|Md|No|Lr)\d.*",
-    "oxides": r".*O\d.*",
-    "nitrides": r".*N\d.*",
-    "sulfides": r".*S\d.*",
-    "halides": r".*[FClBrI]\d.*",
-    "pnictides": r".*[AsSbBi]\d.*",
-    "chalcogenides": r".*[SeTe]\d.*",
-    "borides": r".*B\d.*",
-    "carbides": r".*C\d.*",
-    "hydrides": r".*H\d.*",
-    "oxynitrides": r".*[ON]\d.*",
-}
+split_out_classes = False
+material_classes = (
+    {
+        "all": r".*",
+        "lanthanides": r".*(La|Ce|Pr|Nd|Pm|Sm|Eu|Gd|Tb|Dy|Ho|Er|Tm|Yb|Lu)\d.*",
+        "actinides": r".*(Ac|Th|Pa|U|Np|Pu|Am|Cm|Bk|Cf|Es|Fm|Md|No|Lr)\d.*",
+        "oxides": r".*O\d.*",
+        "nitrides": r".*N\d.*",
+        "sulfides": r".*S\d.*",
+        "halides": r".*[FClBrI]\d.*",
+        "pnictides": r".*[AsSbBi]\d.*",
+        "chalcogenides": r".*[SeTe]\d.*",
+        "borides": r".*B\d.*",
+        "carbides": r".*C\d.*",
+        "hydrides": r".*H\d.*",
+        "oxynitrides": r".*[ON]\d.*",
+    }
+    if split_out_classes
+    else {"all": r".*"}
+)
 n_structs, fig = 200, None
 
+# TODO this includes compliant and non-complaint models
```

Comment:
```suggestion
```

Reply:
addressed in [9a6f5fa](https://github.com/janosh/matbench-discovery/pull/154/commits/9a6f5fa875ad5239a65bdcd252c8facbebf8fffb)